src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/activitiesappointments/ActivitiesAppointmentsClient.kt

```diff
@@ -70,4 +58,21 @@ class ActivitiesAppointmentsClient(private val activitiesAppointmentsApiWebClien
       .retrieve()
       .bodyToMono(AppointmentSeries::class.java)
       .block()
+
+  fun getPrisonersAppointments(prisonCode: String, prisonerNumber: String, onDate: LocalDate): List<AppointmentSearchRequest> =
+    activitiesAppointmentsApiWebClient.post()
```

Comment:
This RETURNS an AppointmentSearchRequest??  Not a Response object?

Reply:
🤔 good spot will check

Reply:
Done